src/main/resources/assets/tis3d/lang/es_es.json

```
@@ -0,0 +1,59 @@
+{
+    "itemGroup.tis3d.name": "TIS-3D",
+
+    "item.tis3d.book_code": "Libro de Codigo",
+    "item.tis3d.book_manual": "Manual de Referencia de TIS-3D",
+    "item.tis3d.key": "Llave",
+    "item.tis3d.key_creative": "Llave Maestra",
+    "item.tis3d.module_audio": "Módulo de Audio",
+    "item.tis3d.module_bundled_redstone": "Módulo Redstone incluido",
+    "item.tis3d.module_display": "Módulo de Pantalla",
+    "item.tis3d.module_execution": "Módulo de Ejecucion",
+    "item.tis3d.module_infrared": "Módulo de Infrarrojos",
+    "item.tis3d.module_keypad": "Módulo de Teclado",
+    "item.tis3d.module_queue": "Módulo de Cola",
+    "item.tis3d.module_random": "Módulo Random",

```

Depending on context here
```suggestion
    "item.tis3d.module_random": "Módulo Randomizador",
```